### PR TITLE
Feature/enroll course service

### PR DESCRIPTION
**Pull request by HoneyHoney483**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline .classpath:1]: Remove .classpath file from feature branch

File: .project [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline .project:1]: remove .project file from feature branch

File: .settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```
Comment (Murutechi) [inline .settings/org.eclipse.core.resources.prefs:1]: Remove .settings file from feature branch

File: src/main/java/com/jm/lms/studentms/controller/EnrollCourseController.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jm.lms.studentms.model.EnrollCourse;
+import com.jm.lms.studentms.service.EnrollCourseService;
+
+@RestController
+@RequestMapping("api/v1/lms/Course")
```
Comment (Murutechi) [inline src/main/java/com/jm/lms/studentms/controller/EnrollCourseController.java:16]: class level uri should be like api/v1/lms/enroll

File: target/classes/application.properties [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Database
```
Comment (Murutechi) [inline target/classes/application.properties:1]: Remove target folder from your feature branch

**Review by Murutechi — CHANGES_REQUESTED** [review]

File: .project (hunk 2 above)
Comment (HoneyHoney483) [reply .project]: Done sir

File: .settings/org.eclipse.core.resources.prefs (hunk 3 above)
Comment (HoneyHoney483) [reply .settings/org.eclipse.core.resources.prefs]: Done sir

File: src/main/java/com/jm/lms/studentms/controller/EnrollCourseController.java (hunk 4 above)
Comment (HoneyHoney483) [reply src/main/java/com/jm/lms/studentms/controller/EnrollCourseController.java:16]: Done sir

**Pull request closed without merging**